CMakeLists.txt

```diff
@@ -24,6 +36,38 @@ set(THIS_PACKAGE_INCLUDE_DEPENDS
     rclcpp
     rviz_visual_tools
     warehouse_ros)
+
+include_directories(include)
+
+# ktopt planning plugin
+add_library(moveit_ktopt_planner_plugin SHARED src/ktopt_planner_manager.cpp
+                                               src/ktopt_planning_context.cpp)
+
+ament_target_dependencies(moveit_ktopt_planner_plugin rclcpp pluginlib
+                          moveit_core moveit_msgs EIGEN3)
+target_link_libraries(moveit_ktopt_planner_plugin drake::drake
+                      ktopt_moveit_parameters)
+
+# Ensure that the plugin finds libdrake.so at runtime
+set_target_properties(
```

Comment:
@kamiradi @sea-bass I needed to set these properties to avoid a segfault when the plugin was loaded because it couldn't find libdrake at runtime. Let me know what you think